images/runtime/node/generateDockerfiles.sh

```diff
@@ -10,7 +10,7 @@ declare -r DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 declare -r DOCKERFILE_TEMPLATE="$DIR/Dockerfile.template"
```

Comment:
![image](https://user-images.githubusercontent.com/1499307/60829698-5a979680-a16a-11e9-89cd-b71924e39773.png)

Reply:
not sure how this happened